src/db.rs

```diff
@@ -5,6 +5,9 @@ use crate::*;
 /// The `sled` embedded database! Implements
 /// `Deref<Target = sled::Tree>` to refer to
 /// a default keyspace / namespace / bucket.
+///
+/// Note that dropping `Db` does not gurantee closing the database. 
+/// Sled may still access the underlying directory under the hood.
```

Comment:
no need for this comment here

Reply:
Where should it be (assuming the comment itself is not wrong, just misplaced)?

Filesystem usage aspect may be important for users, especially, for example, if the same process manages mounts/unmounts of the filesystem that backs the database.